common/decisions/00_major_decisions_british_india.txt

```diff
@@ -31,6 +31,7 @@ form_british_raj_decision = {
 	}
 
 	is_valid = {
+			has_title = title:e_british_raj
```

Comment:
No, as before, we only need this in the `is_shown`.
Reasoning is we don't want to show this to everyone and have them not able to use it because they don't have the British Raj title.

Aka including only in `is_shown` means only someone who completed the first British Raj related decision will see this 'Greater British Raj' decision

Reply:
My original comment said just to remove the wrapper, not also move the code

Reply:
Also I just realised, this has been placed in the wrong decision.